README.md

```diff
@@ -515,6 +515,25 @@ SmalltalkCISpec {
   }
 }
 ```
+The coverage service will default to [coveralls.io][coveralls] to keep backwards compatibility. But you can use:
+- `#coveralls` will use [coveralls.io][coveralls]
+- `#codecov` will use [codecov.io][codecov]
+- `#local` will not attempt to upload the coverage report to any service. Useful for local runs or to setup the upload as a separate step in the build.
+```javascript
+SmalltalkCISpec {
+  ...
+  #testing : {
+    ...
+    #coverage : {
+      #packages : [ 'Packages-To-Cover.*' ],
+      #classes : [ #ClassToCover, #'ClassToCover class' ],
+      #categories : [ 'Categories-To-Cover*' ],
+      #service: #codecov
+    }
+  }
+}
+```
+If the configured service is not `#local`, the `#auto_upload` key will control if the coverage report will be uploaded automatically as part of the test run step or no. In practical terms setting `#auto_upload` to false is equivalent to use `#local` as service.
```

Comment:
> step or no
I don't understand this, is something missing here?

Reply:
I think that probably I want to put "or not" ?